package.json

```diff
@@ -49,7 +49,7 @@
     "css-loader": "^5.0.1",
     "del": "^3.0.0",
     "devtron": "^1.4.0",
-    "electron": "^11.2.1",
```

Comment:
Seems like the newly released electron 11.2.1 causes a crash. We may have to look into this in the future, if the issue is on our side.